Anomaly detection beta (#10752)
DataDog/documentation: content/en/security_platform/explorer.md

- `Related Issues` includes a list of other signals which contain the same group by values to assist with triaging the signal.

### Anomaly detection

> If the Security Signal you are reviewing is generated by the Anomaly Detection method, you will see a graph to visualize the anomaly. On the right hand side of the graph you will see a bounding box displaying where the anomaly was detected.

Reviewer: ```suggestion
If the Security Signal you are reviewing is generated by the Anomaly Detection method, a graph visualizes the anomaly. A bounding box on the right hand side of the graph shows where the anomaly is detected.
```
